docs/Testing/mcp-inspector-testing.md

```diff
@@ -0,0 +1,20 @@
+
+# MCP Inspector Testing Guide
+
+Steps to run and test MCP tools using the https://www.npmjs.com/package/@modelcontextprotocol/inspector.
+
+### 1. **Install MCP Inspector** 
+npx @modelcontextprotocol/inspector
+
+### 2. **Bypass TLS Verification (For Local Testing)**
+set NODE_TLS_REJECT_UNAUTHORIZED=0
+
+### 3. ** Open the inspector with pre-filled token.**
+http://localhost:6274/?MCP_PROXY_AUTH_TOKEN=<token>
+
+### 4. **How to use the tool**
+- Set the transport type "Streamable HTTP".
+- Set the URL "http://localhost:5000/mcp" and hit connect.
```

Comment:
Mention as pre-requisite that dab mcp-server should be running before hitting connect here.